packages/block-editor/src/components/index.js

```diff
@@ -11,7 +11,7 @@ export {
 	BlockAlignmentControl,
 	BlockAlignmentToolbar,
 } from './block-alignment-control';
-export { default as __experimentalBlockFullHeightAligmentControl } from './block-full-height-alignment-control';
+export { default as __experimentalBlockFullHeightAlignmentControl } from './block-full-height-alignment-control';
```

Comment:
I believe this was intentionally exported with a typo in the name for backward compatibility. If you check the export below, it has the correct name.